tests/CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ endif()
 add_subdirectory( XrdOssTests )
 add_subdirectory( XRootD )
 add_subdirectory( cluster )
+add_subdirectory( authenticated_cluster)
```

Comment:
For the sake of getting this in sooner, can we split the addition of this new cluster setup into a separate pull request? As I've discussed with @dynamic-entropy before, the current cluster tests have the annoying issue that it often leaves running processes behind, which then make tests fail the next time you run them. I'd like to improve that and some other things, like using variables for binaries, which can be avoided (like it's done in `tests/XRootD` tests). So ideally we can work together on this and cover also extending the test suite to do tests with Kerberos5 and GSI as well. What do you think?

Reply:
I agree that we should extend the tests to other auth mechanisms too. 

For the issue of failing to run incase of a rerun, the current tests run a cleanup before start, hence helping with the case. One simple way to ensure cleanup is that the script provided to cleanup_fixture does not exit on first error.

As discussed in person, we can sit together and architect what we want going forward. I can simply remove the commit that adds the `authenticated_cluster` to the tests.